nvflare/fuel/hci/client/fl_admin_api_runner.py

```diff
@@ -195,33 +154,5 @@ def run(
             time.sleep(10)  # wait for clients to stop in case they take longer than server to stop
             print("api.check_status(TargetType.CLIENT)")
             api_command_wrapper(self.api.check_status(TargetType.CLIENT))
-            if shutdown_at_end:
-                print("api.shutdown(TargetType.ALL)")
-                api_command_wrapper(self.api.shutdown(TargetType.ALL))
         except RuntimeError as e:
             print(f"There was an exception: {e}")
```

Comment:
so we don't shutdown anymore in the end?

Reply:
shutdown_system still needs implementation work on the server and client side, and shutdown all would not shut down the HA overseer or any other inactive SP